packages/odyssey-react-mui/src/Autocomplete.tsx

```diff
@@ -18,17 +18,19 @@ import {
 import { memo, useCallback } from "react";
 
 import { Field } from "./Field";
+import { FieldComponentProps } from "./FieldComponentProps";
 import type { SeleniumProps } from "./SeleniumProps";
 
+type FieldComponentPropsUsedByAutocomplete = Omit<
+  FieldComponentProps,
+  "isDisabled" | "isReadOnly"
+>;
+
```

Comment:
I don't like `Pick` and `Omit` for this kinda stuff. It just makes it harder to follow what's going on.

Reply:
Is it not possible to disable an `Autocomplete`? And I get that it wouldn't be able to be read-only.

Maybe this is our compromise.

Reply:
re: 
> Is it not possible to disable an Autocomplete?

@KevinGhadyani-Okta `Autocomplete` has a more complex definition of both `isDisabled` and `isReadOnly`. The shared version is a simple `boolean` value. So, didn't seem like I should use those here